### DISCUSSION TITLE
Multiple Users on same set of books?

### DISCUSSION THREAD

**Discussion by StoryHack** [firefly-iii/firefly-iii#5861] [Firefly III]
Is there a way to give a second user access to the same set of books? For example, my wife and I want access one set of books that represesnt our family's finances, and I am the tech guy, so I set everything up, but she likes having her own login account. 

When I open up registration on the admin page, it looks like the new user gets his/her own set of books.

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/#share-your-administration

**Reply by rolders** [reply]
This link seems to be broken - is there an updated answer to this?

**Reply by JC5** [reply]
Yea just use the search.
